source/expressions.tex

```diff
@@ -1475,12 +1488,24 @@
 \begin{codeblock}
 void f() {
   float x, &r = x;
-  [=] {
+  [=]() -> decltype((x)) {      // lambda returns \tcode{float const\&} because this lambda is not \tcode{mutable} and
```

Comment:
```suggestion
  [=]() -> decltype((x)) {      // lambda returns \tcode{\keyword{float} \keyword{const}\&} because this lambda is not \keyword{mutable} and
```